Packages/com.unity.inputsystem/InputSystem/Actions/InputActionRebindingExtensions.cs

```diff
@@ -2347,6 +2344,9 @@ private unsafe void OnEvent(InputEventPtr eventPtr, InputDevice device)
                         continue;
                     }
 
+                    // At this point the control is a potential candidate for rebinding and therefore the event may need to be suppressed, if that's enabled.
```

Comment:
I lack understanding of the full picture here, but given a passing test this probably works as designed.